spalp.module

```diff
@@ -83,42 +82,18 @@ function spalp_get_app_ids(FieldStorageConfig $definition, ContentEntityInterfac
  * Implements hook_modules_installed().
  */
 function spalp_modules_installed($modules) {
+  // TODO: Requires cleanup.
+  // I am not sure this is a good thing!
```

Comment:
why not? performance concerns? it will only be called very rarely

or is it more of an architectural concern? not sure what our other options would be